Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -46,22 +43,7 @@ public struct OfflineMapAreasView: View {
                 Section {
                     preplannedMapAreaViews
                 } header: {
-                    HStack {
-                        Text("Preplanned Map Areas").bold()
-                        Spacer()
-                        Button {
-                            Task {
-                                isReloadingPreplannedMapAreas = true
-                                await mapViewModel.makePreplannedOfflineMapModels()
-                                isReloadingPreplannedMapAreas = false
-                            }
-                        } label: {
-                            Image(systemName: "arrow.clockwise")
-                        }
-                        .controlSize(.mini)
-                        .disabled(isReloadingPreplannedMapAreas)
-                    }
-                    .frame(maxWidth: .infinity)
+                    Text("Preplanned").bold()
                 }
                 .textCase(nil)
```

Comment:
Is there a strong reason not to use the default style for section headers?

![image](https://github.com/user-attachments/assets/b83e8aa6-7c8f-4e9d-838a-a78f40837205)

Actually, do we even need a section header? It is a bit odd to have one since the view itself has a title and there is only one section. Could we get rid of it?

```swift
Section {
    preplannedMapAreaViews
}
```

![image](https://github.com/user-attachments/assets/0dc4f2fd-4a6c-4a3d-8cdd-c178fe1804bc)

Reply:
> Is there a strong reason not to use the default style for section headers?
> 
This change was made previously to be consistent with the section header styling used throughout Apple Maps.

> Actually, do we even need a section header? It is a bit odd to have one since the view itself has a title and there is only one section. Could we get rid of it?
> 
We need a section header to differentiate preplanned from on-demand map areas when we plan to support the on-demand workflow.

Reply:
Okay, thanks. In that case, could you please follow our style of putting each modifier on a single line (or more)?

```swift
Text("Preplanned")
    .bold()
```